doc/INSTALL.md

```diff
@@ -169,6 +169,9 @@ that are not usable on all distributions by default:
   On CentOS it can be necessary to enable this feature with
   `sudo sysctl -w user.max_user_namespaces=10000` or a respective entry
   in `/etc/sysctl.conf` (the exact value is not important).
+  On Ubuntu 24.04 (or newer versions), execute `sysctl -w kernel.apparmor_restrict_unprivileged_userns=0`
+  to enable it. This setting will be reset upon reboot. To apply a persistent setting, create a new file
+  at `/etc/sysctl.d/60-apparmor-namespace.conf` with `kernel.apparmor_restrict_unprivileged_userns=0`.
```

Comment:
Just point to `/etc/sysctl.conf` as above?

For changing `sysctl` values automatically, i.e., via some tool for configuration management, `/etc/sysctl.d` is great. For manual editing by a human it provides little benefit.

Reply:
Like this, or use the same sentence structure as "On CentOS..."?

```markdown
  On Ubuntu 24.04 (or newer versions), execute `sysctl -w kernel.apparmor_restrict_unprivileged_userns=0`
  or add a respective entry in `/etc/sysctl.conf` to enable it.
```

Reply:
Sure, why not the same sentence structure?

Reply:
Maybe I like to be grammatically flexible. Just kidding 😆, I've used the same sentence structure in the next commit.